### PR TITLE
feat(config): add Minoston MP22ZP, fix MP21ZP param 1

### LINKED ISSUE #4627: Minoston MP21ZP: Parameter 1 labeled incorrectly
### Is your problem within Home Assistant (Core or Z-Wave JS Integration)?

NO, my problem is NOT within Home Assistant or the ZWave JS integration

### Is your problem within ZWaveJS2MQTT?

NO, my problem is NOT within ZWaveJS2MQTT

### Checklist

- [X] I have checked the [troubleshooting section](https://zwave-js.github.io/node-zwave-js/#/troubleshooting/index) and my problem is **not** described there.

- [X] I have read the [changelog](https://github.com/zwave-js/node-zwave-js/blob/master/CHANGELOG.md) and my problem was **not** mentioned there.

### Describe the bug

The descriptions for values 0 and 1 of parameter 1, LED Indicator, are inverse from the device behavior. 

Value 0 should read, "On when load is on"
Value 1 should read, "On when load is off"

### Device information

Manufacturer: Minoston
Model name: MP21ZP
Node ID in your network: 37

### How are you using `node-zwave-js`?

- [ ] `zwavejs2mqtt` Docker image (latest)
- [ ] `zwavejs2mqtt` Docker image (dev)
- [ ] `zwavejs2mqtt` Docker manually built _(please specify branches)_
- [ ] `ioBroker.zwave2` adapter _(please specify version)_
- [X] `HomeAssistant zwave_js` integration  _(please specify version)_
- [ ] `pkg`
- [ ] `node-red-contrib-zwave-js` _(please specify version, double click node to find out)_
- [ ] Manually built from GitHub _(please specify branch)_
- [ ] Other _(please describe)_

### Which branches or versions?

version: 0.1.58

### Did you change anything?

no

### If yes, what did you change?

_No response_

### Did this work before?

Don't know, this is a new device

### If yes, where did it work?

_No response_

### Attach Driver Logfile

I don't believe a log file is necessary.

### PR DESCRIPTION
**Pull request by dfederm**
Adds configuration for Minoston MP22ZP, an outdoor smart plug with power meter.

Also fixes #4627

### CODE REVIEW

File: packages/config/config/devices/0x0312/mp22zp.json [hunk 1]
```diff
@@ -0,0 +1,80 @@
+{
+	"manufacturer": "Minoston",
+	"manufacturerId": "0x0312",
+	"label": "MP22ZP",
+	"description": "Z-Wave Outdoor Smart Plug with Power Meter",
+	"devices": [
+		{
+			"productType": "0xff00",
+			"productId": "0xff0f",
+			"zwaveAllianceId": 4669
```
Comment (dfederm) [inline packages/config/config/devices/0x0312/mp22zp.json]: This `zwaveAllianceId` seems [pretty buggy](https://products.z-wavealliance.org/products/4669?selectedFrequencyId=2), but enough fields match up to where I believe it's intended to describe this product despite saying "MP22Z", which is a different product (doesn't have the power meter capabilities). I'll defer to the experts though on whether I should remove the `zwaveAllianceId` or not; all the actual configuration here is based on my physical user manual that came with the products.

File: packages/config/config/devices/0x0312/mp22zp.json [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+		{
+			"#": "8",
+			"$import": "~/0x0312/templates/minoston_template.json#power_meter_reporting_threashold_energy"
+		}
+	],
+	"metadata": {
+		"inclusion": "1. Follow the instructions for your Z-Wave certified controller to add a device to the Z-Wave network.\n2. Once the controller is ready to add your device, press the Manual/Program button on the smart plug 3 time quickly. The blue LED will blink quickly.",
+		"exclusion": "1. Follow the instructions for your Z-Wave certified controller to remove a device to the Z-Wave network.\n2. Once the controller is ready to remove your device, press the manual/program button on the smart plug 3 times quickly.",
+		"reset": "Click the button twice quickly then hold for at least 10 seconds.",
+		"manual": "https://products.z-wavealliance.org/ProductManual/File?folder=&filename=cert_portal/certs/401/marketcert/MP22ZUserManual-1.pdf"
```
Comment (dfederm) [inline packages/config/config/devices/0x0312/mp22zp.json]: As with the `zwaveAllianceId`, this manual link seems to be wrong. It's what's published on the Z-Wave Alliance website though. Again, I'll defer to the experts as to whether I should just remove this instead.

File: packages/config/config/devices/0x0312/mp22zp.json (hunk 1 above)
Comment (dfederm) [reply packages/config/config/devices/0x0312/mp22zp.json]: Considering this is on the website:
> Z-Wave Certification Date: 7/6/2022

And I bought these maybe a month ago, I could be wrong about the `zwaveAllianceId` after all, in which cases there would be no `zwaveAllianceId`.

Deferring to the repo maintainers for guidance.

Comment (dfederm) [reply packages/config/config/devices/0x0312/mp22zp.json]: I removed it; I convinced myself it was wrong. Probably an updated version of the MP22Z product.

File: packages/config/config/devices/0x0312/mp22zp.json (hunk 2 above)
Comment (dfederm) [reply packages/config/config/devices/0x0312/mp22zp.json]: Replaced with a url from the manufacturer's website which matches the physical document I got in the box.

File: packages/config/config/devices/0x0312/mp21zp.json [hunk 3]
```diff
@@ -43,33 +43,15 @@
 		},
 		{
 			"#": "6",
-			"label": "Automating Reporting: Power Interval",
-			"valueSize": 4,
-			"unit": "minutes",
-			"minValue": 1,
-			"maxValue": 54424,
```
Comment (dfederm) [inline packages/config/config/devices/0x0312/mp21zp.json:50]: This seemed like a bug to me. "54424" is "65535" shifted down 1 each decimal digit. I confirmed in the manufacturer's [user manual](https://minoston.com/wp-content/uploads/2021/08/MP21ZP-%E8%AF%B4%E6%98%8E%E4%B9%A6.pdf) that it should be 65535.

File: packages/config/config/devices/0x0312/mp21zp.json [hunk 4]
```diff
@@ -43,33 +43,15 @@
...
-			"defaultValue": 1,
-			"unsigned": true
+			"$import": "~/0x0312/templates/minoston_template.json#power_meter_reporting_threshold_time"
```
Comment (AlCalzone) [inline packages/config/config/devices/0x0312/mp21zp.json]:
```suggestion
			"$import": "templates/minoston_template.json#power_meter_reporting_threshold_time"
```

**Review by AlCalzone — COMMENTED** [review]
I'll do some changes for consistency. Just for future reference, the import paths are relative, unless you prepend them with `~/`

**Review by AlCalzone — APPROVED** [review]

**Pull request merged**